src/brokers/broker-pods/PodsList.container.tsx

```diff
@@ -36,11 +36,22 @@ export const PodsContainer: FC = () => {
   });
 
   const filterBrokerPods = (pods: K8sResourceCommon[], brokerName: string) => {
-    return pods.filter((pod) =>
-      pod.metadata?.ownerReferences?.some(
-        (ref) => ref.name.startsWith(brokerName) && ref.kind === 'StatefulSet',
-      ),
-    );
+    return pods.filter((pod) => {
+      if (!pod.metadata?.ownerReferences) {
+        return false;
```

Comment:
Why has the function shifted from returning a list of pods to a boolean?

Reply:
with my initial implementation i designed to return a list of pods that are associated with a specific broker by using filter method. here the line40 checks if the pod has ownerReferences. If it doesn’t, the function returns false for the selected pod, excluding it from the filtered results.

but now as i am updating with regex pattern i will remove this condition.